BBC Scotland

BrianG61UK opened the pull request:
BBC Scotland SD seems to be on Sky 194 now. BBC Scotland HD is on 876.
This change adds it to the list for HD swap.

Please DO carefully check my work.

Thank you.

AbuBaniaz commented:
Can you remove the Bloomberg HD swap line please? There is no SD version, so that swap isn't required.

BrianG61UK commented:
l left the Bloomberg line in because it was originally there.

I did assume it was not needed anymore.
But I added a condition to make sure it wouldn't be a problem instead of removing it.

I've removed it now.

Thanks for your help.
